docs/Documentation/Scripting/Building-Blocks/Integration-List.md

```diff
@@ -1128,3 +1128,5 @@ To complete this objective you need to enter WorldGuard region with specified na
 objectives:
   deathZone: "region deathZone entry events:kill"
 ```
+## [RedisChat](https://emibergo.gitbook.io/redischat/)
```

Comment:
If you scroll compleatly up here, you find a list with all integrations, where you also need to add RedisChat

Reply:
Ok done